graph/protocol.go

```diff
@@ -186,6 +207,35 @@ func addToMetadataValue(md map[string]interface{}, k string, v float64) {
 	}
 }
 
+// The metadata for response codes is a map of maps. Each response code is broken down by responseFlags:percentageOfTraffic like this:
+// "200" : {
+//     "-"  : 90.0,
+//	 "DC" : 10.0,
```

Comment:
this being indented further than the "-" line is just a typo right? There is no second level of data - I think "-" and "DC" are peers here? But the comment makes it seem like "DC" is indented (under)  "-".

Reply:
Ooh, a tab in there, yuk.  Will fix.